src/service/resources-impl.js

```diff
@@ -602,7 +602,7 @@ export class ResourcesImpl {
           'sourceUrl': getSourceUrl(this.ampdoc.getUrl()),
           'serverLayout': doc.documentElement.hasAttribute('i-amphtml-element'),
           'linkRels': documentInfo.linkRels,
-          'metaTags': documentInfo.metaTags,
+          'viewport': documentInfo.viewport,
```

Comment:
Ah, sorry I meant we still need `metaTags` in the message contents, but it's OK if it only contains `meta[name=viewport]`. 

I.e. I think this should be:
```js
'metaTags': {'viewport': documentInfo.viewport},
'viewport': documentInfo.viewport,
```

Feel free to drop a TODO (or new issue) to rename this once the callers have been migrated to use `viewport` instead of `metaTags`.

Reply:
I'm happy to do this, but I'm curious where this request to retain `DocumentInfo.metaTags['viewport']` is coming from? After this PR, if you grep for `metaTags` in this repository, you won't find a single hit. So apparently there's a consumer of this API outside of AMPHTML?